src/main/components/RequestBar.jsx

```diff
@@ -39,14 +40,39 @@ const RequestBar = (props) => {
     e.preventDefault();
   };
 
+  // Extra fetches for performance metrics
+  const fetchTimesList = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0];
```

Comment:
This is for Sample data?  We can discus more as a team as to how the number is determined. But this is ok for now the tool works great.